README.md

```diff
@@ -22,8 +22,9 @@ The following parameters should be set:
 ```yaml
 ---
 :enabled: true
-#:pulp_url: https://localhost/
-#:content_app_url: https://localhost:24816/
+:pulp_url: https://foreman.example.com/
+:content_app_url: https://foreman.example.com/pulp/content
+:rhsm_url: https://foreman.example.com/rhsm/
```

Comment:
Nit: I'd prefer `pulp.example.com` since we don't mandate they're on the same server.